setup.cfg

```diff
@@ -20,7 +20,7 @@ setup_requires =
 # Specify any package dependencies below.
 install_requires =
     networkx>=2.0
-    numpy
+    numpy<2.0 # See https://github.com/mdavidsaver/p4p/issues/145
```

Comment:
Can we put this pinning in the pva section?

Now that we have split ophyd async out do we need p4p as a dep anymore?

Reply:
Yeah that makes sense, I'll move it there. I don't know whether you need the pva section at all anymore but do feel like it's a bit of a separate issue to this.

Reply:
Opened a separate issue for removing the PVA section: https://github.com/bluesky/ophyd/issues/1201.